tests/Toolkit/ControllerTest.php

```diff
@@ -29,6 +29,20 @@ public function testArguments()
 		]));
 	}
 
+	/**
+	 * @covers ::arguments
+	 */
+	public function testVariadicArguments()
+	{
+		$controller = new Controller(fn ($c, ...$args) => $c . implode('', $args));
```

Comment:
To ensure that `$c` still receives its value correctly:

```suggestion
		$controller = new Controller(fn ($c, ...$args) => $c . '/' . implode('', $args));
```